package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "ng": "nx",
     "nx": "nx",
-    "start": "ng serve",
+    "start": "ng serve && npm run build:core:watch",
```

Comment:
This is not going to work - ng serve does not terminate, so core is never built. 

Consider concurrently, npm-run-all or similar to run the processes besides each other.